imports/ui/components/task/task.html

```diff
@@ -1,11 +1,10 @@
 <template name="task">
   <li class="{{#if task.checked}}checked{{/if}}">
-    <button class="delete">&times;</button>
-
+    
     <input type="checkbox" checked="{{task.checked}}" class="toggle-checked" />
```

Comment:
The reason you're getting an error when you check a task is because you're trying to directly modify an attribute here with {{task.checked}}. You'll want to have a helper method on the template prob, and then make sure the server understands what you're trying to do.